openupgrade_scripts/scripts/point_of_sale/16.0.1.0.1/pre-migration.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2024 Tecnativa - Pedro M. Baeza
+# SPDX-License-Identifier: AGPL-3.0-or-later
+import uuid
+
+from openupgradelib import openupgrade
+
+
+def _assign_pos_order_token(env):
+    openupgrade.add_fields(
+        env,
+        [("access_token", "pos.order", "pos_order", "char", False, "point_of_sale")],
+    )
+    env.cr.execute("SELECT id FROM survey_survey")
+    for row in env.cr.fetchall():
+        env.cr.execute(
+            "UPDATE survey_survey SET access_token = %s WHERE id = %s",
+            (str(uuid.uuid4()), row[0]),
+        )
```

Comment:
I installed point of sale in a V15 database and a V16 database, and in each case, the request fail ``SELECT id FROM survey_survey;``
ERREUR:  la relation « survey_survey » n'existe pas
LIGNE 1 : SELECT id FROM survey_survey;

That is quite normal as survey.survey is defined in survey module that is not a dependency of point_of_sale module.

1) Could you elaborate why you introduced that code ? 
2) I can not understand why the CI is green.

Reply:
Fixed in https://github.com/OCA/OpenUpgrade/commit/dea8b8a70af3e9b3cc29a5b472d2699b0ba8f212 and merging `pos_sale` for normalizing CI.